en-us/docs/developers/user-guide/faq_dev.html

```diff
@@ -25,7 +25,7 @@ <h3>Which version should I choose?</h3>
 <p>2.6.x: requires Java 1.6, minor feature &amp; bugfix branch, GA, production ready.</p>
 </li>
 <li>
-<p>2.5.x: requires Java 1.6, maintenance branch, only accept security vulnerability and critical bugfix, expected to be EOL soon.</p>
+<p>2.5.x: requires Java 1.6, maintenance branch, only accept security vulnerability and critical bugfix, the support will end on 2019-6-15, please upgrade to 2.6.x as soon as possible.</p>
```

Comment:
Why these lines are changed? Could you please rebase your branch with the current master?

Reply:
done